parquet-format-structures/src/main/java/org/apache/parquet/format/Util.java

```diff
@@ -130,7 +131,7 @@ public static PageHeader readPageHeader(InputStream from) throws IOException {
 
   public static PageHeader readPageHeader(InputStream from, 
       BlockCipher.Decryptor decryptor, byte[] AAD) throws IOException {
-    return read(from, new PageHeader(), decryptor, AAD);
+    return validate(read(from, new PageHeader(), decryptor, AAD));
```

Comment:
I think it would be more clear if you called `MetadataValidator.validate` rather than just `validate`.